public/app/features/dashboard-scene/scene/LibraryPanelBehavior.tsx

```diff
@@ -50,6 +51,10 @@ export class LibraryPanelBehavior extends SceneObjectBase<LibraryPanelBehaviorSt
 
     const libPanelModel = new PanelModel(libPanel.model);
 
+    // Use dashboard panel ID for data layer filtering
+    // const dashboardPanelId = getPanelIdForVizPanel(vizPanel);
+    // libPanelModel.id = dashboardPanelId;
```

Comment:
Should these lines be commented out?

Reply:
Lol, I was testing something locally and ended up commenting out these lines, good catch!